src/components/searchBar.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+
+class searchBar extends Component {
+  state = {
+    input: '',
+  };
+  handleChange = (event) => {
+    const { value } = event.target;  
+    this.props.filteredFood(event.target.value);
```

Comment:
the component is working correctly. However, here you are passing event.target.value. In line 8 you deconstructed the value from event.target, so in this line you can only put value and it will work.